.golangci.yml

```diff
@@ -2,7 +2,7 @@ run:
   timeout: 5m
   tests: false
   skip-dirs:
-    - client/v2/common
+    - client/v2
```

Comment:
Is there a new warning that precipitated increasing the skip?

Reply:
Yes, this directory contains all the generated files (including ones from this PR in `algod.go`). It would be nice to change the generator to generate lint-compliant Go code, but I figured it was out of scope for now.